src/main/java/org/datadog/jenkins/plugins/datadog/clients/DatadogHttpClient.java

```diff
@@ -622,20 +770,50 @@ private String getJenkinsVersion(){
 
     @Override
     public boolean startBuildTrace(BuildData buildData, Run run) {
-        logger.fine("Traces are only available using Datadog Agent.");
-        return false;
+        if(this.isWebhookIntakeConnectionBroken()){
+            logger.severe("Your client is not initialized properly");
```

Comment:
```suggestion
            logger.severe("Unable to start build trace; your client is not initialized properly.");
```
nit to make debugging easier